dispatch/modules/content/models.py

```diff
@@ -302,14 +302,23 @@ def save_topic(self, topic_id):
             except Topic.DoesNotExist:
                 pass
 
+    def get_authors(self):
+        return Author.objects.filter(article=self)
+
     def save_authors(self, authors):
         # Clear current authors
         Author.objects.filter(article=self).delete()
 
         # Create a new author for each person in list
         # Use `n` to save authors in correct order
-        for n, person_id in enumerate(authors):
-            Author.objects.create(article=self, person_id=person_id, order=n)
+        for n, author in enumerate(authors):
+            Author.objects.create(
+                article = self,
+                person_id = author['person["id"]'],
```

Comment:
This is close! If you remove the `person` argument and use `person_id=author['person']` it should work. It's a bit counterintuitive, I know.

`person_id` expects an integer ID and `person` expects an instance of a person. In this case, `author['person']` is an ID and not an instance, so we have to use the `person_id` argument. Both accomplish the same thing, though.